pkg/utils/helpers.go

```diff
@@ -46,6 +46,15 @@ func TruncateString(in string, num int) string {
 	return out
 }
 
+func Remove(s []string, r string) []string {
+	for i, v := range s {
+		if v == r {
+			return append(s[:i], s[i+1:]...)
```

Comment:
if `i == len(s)` i.e. the last element this would panic with an out of bounds

Reply:
That's not true, Because `i + 1 = len(s)` which means that `s[i+1:] = s[len(s):] = []string{}`

Reply:
Still added the test cases because we definitely need the code coverage 😃